### ISSUE TITLE
Missing cofig

### ISSUE THREAD

**Issue by xingyaoshawn** [DiscourseVR/Discourse-WebClient-Vue#1]
Hi,
I'm reproducing your project, but it seems that the reference of API_URL from the file config shows an error.
I assume that there should be a file config but not included in this file.
for example, in the file videos.service.js : import { API_URL } from '../config'

**Comment by fairnightzz** [comment]
Correct, you need to supply ur own api url. Why would we give you our api url 🗿

**Comment by MrPicklePinosaur** [comment]
API_URl should be set as the root of the server you are serving the files from. For us this was a subdomain pointing to an nginx autoindex page. You can also see in the rest of the `videos.service.js` file we are also expecting there to be a `/vids` directory that the videos will be served from.
